roles/gateway_configure_hsm_luna/tasks/main.yml

```diff
@@ -82,8 +87,11 @@
 
 - name: Log in to the Luna HSM appliance and register the client
   register: clientreg
-  command: sshpass -p '{{ hsm_password }}' ssh {{ ansible_ssh_extra_args }} {{ hsm_usr }}@{{ hsm_hostname }} client register -client {{ inventory_hostname }} -hostname {{ inventory_hostname }}
+  command: >
```

Comment:
Just curious what it means to have command: >

Reply:
It is YAML syntax for a “Folded Block Scalar” > . It will fold newlines here to spaces, so at runtime ansible will read it as a one-line command

Reply:
It is similar to the “Literal Block Scalar” `|` which is used to write literal multi-line values e.g. we use those in vault files